test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/Infrastructure/ApplicationTestFixture.cs

```diff
@@ -63,8 +63,8 @@ public virtual DeploymentParameters GetDeploymentParameters()
                 RuntimeArchitecture.x64)
             {
                 PublishApplicationBeforeDeployment = true,
-#if NETCOREAPP1_1
-                TargetFramework = "netcoreapp1.1",
+#if NETCOREAPP2_0
```

Comment:
Think we can remove the `ifdef`, we don't compile for another TFM here and don't plan on doing that